integration/data/common.py

```diff
@@ -54,6 +54,45 @@ def _base():
     return path.dirname(__file__)
 
 
+def get_dev(replica1, replica2, controller):
+    prepare_backup_dir(BACKUP_DIR)
+    open_replica(replica1)
+    open_replica(replica2)
+
+    replicas = controller.list_replica()
```

Comment:
Duplicate with fixture function dev() below. Refactor dev() to call to get_dev() instead.